Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -52,17 +52,46 @@ public function provideCases()
 ',
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;",
+                "<?php\r\$a = 4;\n",
+                "<?php\r\$a = 4;",
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;\r",
+                "<?php\r\$a = 5;\n",
+                "<?php\r\$a = 5;\r",
```

Comment:
Maybe we shouldn't, so your issue @ https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1092 is still valid. 
However this is not the issue with this fixer I try to resolve with this PR. I rather keep this PR atomic; so I'm sorry but I won't fix the other issue. Please note that although this PR doesn't solve the your issue it doesn't introduce it either.